studio/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:14
```

Comment:
Any reason we're not using the latest LTS? :slightly_smiling_face: (1/2)

```suggestion
# Node v16.14.0 LTS
FROM node@sha256:fd86131ddf8e0faa8ba7a3e49b6bf571745946e663e4065f3bff0a07204c1dde
```

Reply:
I figured there'd be a reason! Ok, makes good sense to me. :slightly_smiling_face: